client/src/components/ReviewFilters/ReviewTypeFilter.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { ReviewTypeContainer, ReviewTypeTitle, ReviewTypeFlyoutMenu, DownArrow, FlyoutMenuText, FlyoutMenuStat } from '../../styled';
+
+const ReviewTypeFilter = ({ reviewStats, updateReviewFilters }) => (
+  <ReviewTypeContainer>
+    <ReviewTypeTitle>
+      Review Type
+      <DownArrow className="review_type_down_arrow">&#9662;</DownArrow>
+      <ReviewTypeFlyoutMenu className="review_type_flyout">
+        <form>
+          <input type="radio" value="all" name="reviewType" className="all_reviews_filter" onChange={(e) => { updateReviewFilters(e.target.value, 'reviewType'); }} />
```

Comment:
It looks like the input tag is reused multiple times in the same way. Did you think about breaking that out into a styled component? You can actually specify attributes within styled-components so the ones that are shared only have to be put in one place and will only have to type out the name attribute where they differ and free up some real estate.  It looks like this.
![Screen Shot 2021-02-06 at 9 09 50 PM](https://user-images.githubusercontent.com/66852739/107134435-0b729400-68c0-11eb-9b6c-39e67d541c6c.png)